core/src/main/scala/org/apache/spark/Heartbeater.scala

```diff
@@ -59,12 +60,13 @@ private[spark] class Heartbeater(
     heartbeater.awaitTermination(10, TimeUnit.SECONDS)
   }
 
-  /**
-   * Get the current executor level metrics. These are returned as an array, with the index
-   * determined by MetricGetter.values
-   */
+  /** Get the current executor level metrics. These are returned as a Map */
   def getCurrentMetrics(): ExecutorMetrics = {
-    val metrics = ExecutorMetricType.values.map(_.getMetricValue(memoryManager)).toArray
+    // figure out how to append all the metrics
```

Comment:
this is a strange comment ... is this a TODO?

Reply:
No the commit should have been removed. will fix it. Thanks